internal/flink/command_detached_savepoint_delete.go

```diff
@@ -0,0 +1,50 @@
+package flink
+
+import (
+	"github.com/spf13/cobra"
+
+	pcmd "github.com/confluentinc/cli/v4/pkg/cmd"
+	"github.com/confluentinc/cli/v4/pkg/deletion"
+	"github.com/confluentinc/cli/v4/pkg/errors"
+	"github.com/confluentinc/cli/v4/pkg/resource"
+)
+
+func (c *command) newDetachedSavepointDeleteCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "delete <name>",
+		Short: "Delete Flink detached savepoint in Confluent Platform.",
+		Args:  cobra.MaximumNArgs(1),
+		RunE:  c.detachedSavepointDelete,
+	}
+	addCmfFlagSet(cmd)
+	cmd.Flags().Bool("force", false, "Force delete the savepoint.")
```

Comment:
```suggestion
	pcmd.AddForceFlag(cmd)
```
We can just use the regular flag wording for this.